packages/components/tree-select/src/tree.ts

```diff
@@ -121,6 +121,13 @@ export const useTree = (
     })
   }
 
+  const emitModelChange = (value) => {
```

Comment:
It looks good.

Just by looking at the previous writing style, I think we can divide it into two functions.

https://github.com/element-plus/element-plus/blob/395af6672a15c5470b23a03020b8a6c669665c12/packages/components/select-v2/src/useSelect.ts#L511

```ts
const emitChange = (val: any | any[]) => {
  if (!isEqual(props.modelValue, val)) {
    emit(CHANGE_EVENT, val)
  }
}

function update(val) {
	emit(UPDATE_MODEL_EVENT, val)
  emitChange(val)
}
```